streaming/nifi/data.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks_blueprints.eks_cluster_id
+}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_region" "current" {}
+
+data "aws_secretsmanager_secret_version" "admin_password_version" {
+  secret_id = aws_secretsmanager_secret.grafana.id
+
+  depends_on = [aws_secretsmanager_secret_version.grafana]
```

Comment:
I don't think this depends_on is required here since the dependency is implied by the use for `aws_secretsmanager_secret.grafana.id`